kotlinx-coroutines-core/jvm/test/TestBase.kt

```diff
@@ -254,3 +254,10 @@ public actual open class TestBase(private var disableOutCheck: Boolean)  {
 
     protected suspend fun currentDispatcher() = coroutineContext[ContinuationInterceptor]!!
 }
+
+/*
+ * We ignore tests that test **real** non-virtualized tests with time on Windows, because
+ * our TC Windows is virtualized itself (oh, the irony) and its clock resolution is dozens of ms,
```

Comment:
```suggestion
 * our CI Windows is virtualized itself (oh, the irony) and its clock resolution is dozens of ms,
```
Should be more clear to external contributors.